analyses/focal-cn-file-preparation/04-prepare-cn-file.R

```diff
@@ -260,11 +261,17 @@ if (xy_flag) {
   # Removing copy neutral segments saves on the RAM required to run this step
   # and file size
   cnv_sex_chrom <- cnv_df %>%
-    dplyr::filter(chr %in% c("chrX", "chrY"), status != "neutral")
+    dplyr::filter(chr %in% c("chrX", "chrY"))
 
   # Merge and annotated no X&Y
   sex_chrom_annotated_cn <- process_annotate_overlaps(cnv_df = cnv_sex_chrom,
-                                                      txdb_exons = tx_exons)
+                                                      txdb_exons = tx_exons) %>%
+    # mark possible deep loss in sex chromosome
+    dplyr::mutate(status = dplyr::case_when(
+      copy_number == 0  ~ "deep deletion",
+      TRUE ~ as.character(status)
+    ))
```

Comment:
I think we will come back to this, as I think due to the nature of the sex chr not always being diploid (XY), the losses are capturing broad and not just focal losses, so adding a note here.